test/Kestrel.Core.Tests/Http2ConnectionTests.cs

```diff
@@ -2061,6 +2061,17 @@ public Task HEADERS_Received_HeaderBlockContainsTEHeader_ValueIsNotTrailers_Stre
 
             await SendGoAwayAsync();
 
+            var spinLimit = 50; // 2.5 seconds
+            while (_connection.State == Http2Connection.ConnectionState.Open && spinLimit --> 0)
```

Comment:
Is that `spinLimit-- > 0`?

Reply:
yes but I liked the aesthetics of the "goes to" operator